README.md

```diff
@@ -85,9 +88,10 @@ mic = PyAudioInput()
 vad = VoiceActivityDetector()
 wake = WakewordTrigger("path_to_tflite_model")
 asr = SpeechRecognizer("spokestack_id", "spokestack_secret")
+timeout = ActivationTimeout()
 
 
-pipeline = SpeechPipeline(mic, [vad, wake, asr])
+pipeline = SpeechPipeline(mic, [vad, wake, asr, timeout])
```

Comment:
In newer versions, the `start()` call below isn't needed, right?

Reply:
yeah I can remove that also